decidim-accountability/app/assets/javascripts/decidim/accountability/admin/accountability_admin.js.es6

```diff
@@ -12,34 +12,34 @@ $(function() {
   $(document).on("open.zf.reveal", "#data_picker-modal", function () {
     let xhr = null;
 
-    $('#data_picker-autocomplete').autoComplete({
+    $("#data_picker-autocomplete").autoComplete({
       minChars: 2,
       source: function(term, response) {
         try {
           xhr.abort();
         } catch (exception) { xhr = null}
 
         xhr = $.getJSON(
-          'proposals.json',
+          "proposals.json",
           { term: term },
           function(data) { response(data); }
         );
       },
       renderItem: function (item, search) {
-        let sanitizedSearch = search.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
-        let re = new RegExp(`(${sanitizedSearch.split(' ').join('|')})`, "gi");
+        let sanitizedSearch = search.replace(/[-/\\^$*+?.()|[\]{}]/g, "\\$&");
```

Comment:
This thingies are equivalent right? I mean, escape character interpretation is not influenced by the type of quoting... or is it?